snap_env

```diff
@@ -92,6 +92,17 @@ while [ -z "$SETUP_DONE" ]; do
     echo "Vivado version          is set to: `vivado -version|head -n1`"
   fi
 
+  ####### checking default variables
```

Comment:
I'd only add the lines that are required for the selected CAPI version

Reply:
ok correction done to set only the variable needed for the type of CAPI